arbor/morph/locset.cpp

```diff
@@ -250,16 +229,13 @@ locset most_proximal(region reg) {
 }
 
 mlocation_list thingify_(const most_proximal_& n, const mprovider& p) {
-    auto extent = thingify(n.reg, p);
```

Comment:
Why remove the assertions here?

Reply:
Because it isn't asserting preconditions on the output of this function: it was asserting the output of the `thingify` code. This wasn't done anywhere else in the region or locset code, and we have unit tests that verify that `thingify` calls are valid.

And it made the code more complex!